uktrade/data-hub-frontend
# Accessibility: Fix  text contrast errors on certain elements

## Description of change

As part of the Accessibility work, a small sample of pages have been tested for Accessibility issues before we get them audited. This PR focuses on changing the colours and styles of some elements which flagged up as not having a significant enough text contrast. 

The elements that have been amended are: 

- The hint text on the collection filters that sit on the right hand side of the Companies (and other) pages. It has been darkened to stand out more against it's grey background.

- The Active filters that sit in the collection header in the Companies (and other) pages. It has been made to look a touch more like the GOVUK filter tags, with a thin black border, a light grey background, and had the tag label text darkened so that it stands out more. 

## Test instructions

You should be able to navigate to any of the tabs within the grey navigation banner, and see that the hint text in the filters is #505A5F. You should also be able to navigate to the companies page and apply a filter. There you will see it's new design of a background of #F3F2F1, and a 1px wide solid border. 

The visual tests have been updated for this PR, however please pay attention to these as It's the first time I've run / updated visual tests! 

## Screenshots

Filter hint text before: 

<img width="306" alt="Screenshot 2020-09-15 at 16 57 48" src="https://user-images.githubusercontent.com/46787754/93754902-56825800-fbfa-11ea-93aa-ab024e889e76.png">


Filter hint text after: 

<img width="330" alt="Screenshot 2020-09-15 at 16 46 28" src="https://user-images.githubusercontent.com/46787754/93754992-71ed6300-fbfa-11ea-89bb-77120661bab1.png">


Filter tags before: 

<img width="677" alt="Screenshot 2020-09-15 at 16 57 55" src="https://user-images.githubusercontent.com/46787754/93755032-8a5d7d80-fbfa-11ea-9374-ac80f4c8758f.png">


Filter tags after: 

<img width="659" alt="Screenshot 2020-09-21 at 11 51 32" src="https://user-images.githubusercontent.com/46787754/93758694-d0b5db00-fc00-11ea-849a-b3401bb783cf.png">


Interaction design requested that we try to make the filter tags look closer to the GOV.UK ones like so: 

<img width="292" alt="Screenshot 2020-09-21 at 11 26 21" src="https://user-images.githubusercontent.com/46787754/93756783-8c750b80-fbfd-11ea-977d-8d4e1631c3bf.png">




## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
